### PR TITLE
[WINMINE] Add sound effects

### PR DESCRIPTION
**Pull request by TAN-Gaming**
## Purpose
Add sound effects to the WineMine game.

## TODO
- [x] Allow turning on/off (sound is disabled by default, use Windows-compatible registry key if possible)

### CODE REVIEW

File: base/applications/games/winmine/main.c [hunk 1]
```diff
@@ -653,15 +663,19 @@ static void CompleteBox( BOARD *p_board, unsigned col, unsigned row )
...
             p_board->face_bmp = DEAD_BMP;
             p_board->status = GAMEOVER;
+            PlaySoundEffect(IDW_EXPLODE);
         }
-        else if( p_board->status != GAMEOVER )
+        else if( p_board->status != GAMEOVER ) {
             p_board->boxes_left--;
+            if (is_first_box)
+                PlaySoundEffect(IDW_BOX);
+        }
```
Comment (binarymaster) [inline base/applications/games/winmine/main.c:672]: The code is winesynced, except the resources: https://github.com/reactos/reactos/blob/c3ec1b9ac63ec628f49d8e10c0c1d7950bb26c5a/media/doc/WINESYNC.txt#L248

Comment (cbialorucki) [reply base/applications/games/winmine/main.c:672]: I doubt Wine maintains this anymore, however. Same with WordPad and other little utilities like this.
If someone really wanted to fork it, I'd approve it; but I don't see the value in doing that at the moment.

Comment (TAN-Gaming) [reply base/applications/games/winmine/main.c]: Fixed.

**Comment by TAN-Gaming** [comment]
https://github.com/reactos/reactos/assets/86486978/75c505b6-024c-47fc-a25f-5ef7026225ce

**Review by binarymaster — APPROVED** [review]

**Review by Doug-Lyons — APPROVED** [review]

**Review by oleg-dubinskiy — APPROVED** [review]

**Pull request merged**